src/state/bridge/hooks.ts

```diff
@@ -197,3 +221,22 @@ export function useCalculatedBridgeFee(tokenAddress: string | undefined, currenc
     }
   }, [isHome, account, amount, library, tokenAddress])
 }
+
+export function useDetectBridgeDirection(selectedBridgeDirection?: BridgeDirection) {
+  const { chainId } = useActiveWeb3React()
+
+  if (selectedBridgeDirection) {
+    return selectedBridgeDirection
+  }
+
+  switch (chainId) {
+    case ChainId.FUSE:
+      return BridgeDirection.FUSE_TO_ETH
```

Comment:
`BridgeDirection.FUSE_TO_ETH` doesn't sound correct, cause it can be also to BSC

Reply:
i thought it would be good to add a default, i have an alternative in mind maybe i can implement this, which would show an error on the button
```suggestion
      if (isHome && !bridgeDirection) {
        inputError = inputError ?? 'Select Destination'
      }
```